compiler/resolution/Makefile.share

```diff
@@ -1,29 +1,30 @@
 # Copyright 2004-2016 Cray Inc.
 # Other additional copyright holders may be indicated within.
-# 
+#
 # The entirety of this work is licensed under the Apache License,
 # Version 2.0 (the "License"); you may not use this file except
 # in compliance with the License.
-# 
+#
 # You may obtain a copy of the License at
-# 
+#
 #     http://www.apache.org/licenses/LICENSE-2.0
-# 
+#
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-RESOLUTION_SRCS = \
-	caches.cpp \
-	callDestructors.cpp \
-	callInfo.cpp \
-	cullOverReferences.cpp \
-	implementForallIntents.cpp \
-	generics.cpp \
-	functionResolution.cpp \
-	lowerIterators.cpp \
+RESOLUTION_SRCS =                                    \
+        addAutoDestroyCalls.cpp                      \
+	caches.cpp                                   \
```

Comment:
Not sure if you fixed this later, but I think these should remain tabs instead of spaces.